src/app/_services/auth.service.ts

```diff
@@ -433,11 +434,12 @@ export class AuthService implements CanActivate {
    */
   public showPostLoginBanner(): boolean {
     let user = this.getUser()
+    let showBanner = user.preference.showAJIModalAndBanner ? user.preference.hasOwnProperty('showAJIModalAndBanner') : false
```

Comment:
I don't think we need to check this again here.

Reply:
Not sure if I follow correctly. Do we want to let go of the check entirely?